lib/puppet/provider/sensuctl_configure/sensuctl.rb

```diff
@@ -66,6 +66,16 @@ def config_namespace=(value)
     @property_flush[:config_namespace] = value
   end
 
+  def backend_init
```

Comment:
is this temporary, ie is it only for 5.16 and will go away with another version that has the return code?

Reply:
No, this is permanent. After some testing the return code won't help because we still have to execute `init` to get the code.